sonar-plugin/src/main/resources/io/ecocode/profiles/ecocode_typescript_profile.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "ecoCode",
   "ruleKeys": [
-    "EC13"
+    "EC13",
+    "EC36"
```

Comment:
as the rule is both for JavaScript and TypeScript, you don't need to put it here (TypeScript-only rules)